darasimiolaifa/amistad
# chore(frontend): setup frontend of app

## What does this PR do?
This PR sets up the front end part of the project

## What major activities were carried out?
- create a react app using create-react-app
- remove unneccessary files and code from the project
- install react-router-dom and materialize-ui
- create the home, signup and login pages
- create the navbar component
- write code to show landing page and navigation

## How can this be manually tested?
- Clone the repository
- Fetch the branch
- Install the dependencies
- cd into the `amistad-client` folder
- Start project by running `npm start` in the terminal
- Your browser should automatically load to show you the homepage screen below:

![homepage](https://user-images.githubusercontent.com/28527393/71343617-b0b59600-2560-11ea-9729-fff627cfefd9.png)
- Click on the `Signup` button, your scree should change the one below:

![signup](https://user-images.githubusercontent.com/28527393/71343692-de9ada80-2560-11ea-8a44-580746405a54.png)
- Click on the `Login` button, your scree should change to the one below:

![login](https://user-images.githubusercontent.com/28527393/71343745-00945d00-2561-11ea-8e0a-382cad3a6a68.png)
